api/inbox.go

```diff
@@ -16,14 +17,40 @@ func (w Wrapper) GetTransferRequest(ctx echo.Context, requestorDID string, fhirT
 	if err != nil {
 		return err
 	}
-	transferRequest, err := w.TransferReceiverService.GetTransferRequest(ctx.Request().Context(), cid, requestorDID, fhirTaskID)
+
+	var sessionWithUserContext *Session
+
+	for _, session := range w.APIAuth.GetSessions() {
```

Comment:
Ik looks like you just get the fist session with a user context? When multiple people are logged in with an elevated session, don't you rist mixing them up?

Reply:
Ah yes indeed. Will fix ;)